bemb/model/bemb.py

```diff
@@ -76,10 +81,16 @@ def is_coefficient(name: str) -> bool:
     def is_observable(name: str) -> bool:
         return any(name.startswith(prefix) for prefix in observable_prefix)
 
+    utility_string = utility_string.replace(' - ', ' + -')
```

Comment:
We might want to find a way to parse utilities even when the user does not put spaces around `+` or `-`; let's do this later.

Reply:
agree